styles/style.css

```diff
@@ -0,0 +1,108 @@
+/* Normalize */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* MEDIA QUERY MOBILE VERSION */
+
+@media screen and (min-width: 375px) and (max-width: 768px) {
```

Comment:
- [ ] **[OPTIONAL]:** I'm sorry that the previous reviewer didn't point this out, but when you're working on a mobile-first approach, you're assuming that the global styles apply for mobile screens, and you add media queries only when you want to make your design responsive into bigger screens. 📜 So you don't need this media query at all, your styles will work fine until you need to make your design responsive for desktop screens which will happen later. I strongly suggest you remove this. 🤓 🔧